### ISSUE TITLE
 Macro names and parameters should not be visible

### ISSUE THREAD

**Issue by rasmus-pagh** [arXiv/html_feedback#253]
### Description

 Macro name and parameters are not visible in the rendered PDF file, but only in the HTML version.
 This problem seems to arise for all paper using the standard class for the TheoretiCS journal (https://theoretics.episciences.org/)

![Screenshot 2024-01-08 at 18 03 11](https://github.com/arXiv/html_feedback/assets/16596230/8f62f579-b895-45a4-a0d1-de9e5bb02c40)

If this can not be changed on arXiv's end, the editors of TheoretiCS would be interested in how to change the class file to avoid the issue.

### (Required) What is this issue most closely related to? Select one.

Other

### Internal issue ID

141f2647-0033-4042-a40c-815bb7f16acc

### Paper URL

https://browse.arxiv.org/html/2212.12802v2

### Browser

Chrome/120.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
@rasmus-pagh it's not about changing the original class file per se. Instead, one would need a new binding (a `.cls.ltxml` file) contributed to the LaTeXML project, so that the macros receive appropriate support.

**Comment by rasmus-pagh** [comment]
Thanks for the quick reply! The journal is nonprofit, open access, and run by volunteers so I am not sure if there are resources for them to create a mapping file. On the other hand it is potentially not too complicated in which case I might be able to help.

**Comment by dginev** [comment]
Right, it's either their team, some of their authors, or us (the latexml team) that eventually has to spend some time and contribute a mapping file.

We have some last resort fallbacks in place in latexml, where an unknown class may receive partial support, but only when the frontmatter macros defined by the class file match exactly the standard names (and the usual argument signatures). If `\ThCSauthor` was exposed as the regular `\author`, the fallback support would have kicked in.

But if you'd like high quality markup, someone has to write a dedicated mapping file.
